webapp/e2e/package.json

```diff
@@ -20,6 +21,7 @@
     "lodash.without": "4.4.0",
     "lodash.xor": "4.5.0",
     "mattermost-redux": "5.26.2",
+    "mattermost-webapp": "github:mattermost/mattermost-webapp#40853eab863f321bd64d67efe1b986864b01278c",
```

Comment:
I changed the webapp repo commit to be before the `E2EClient` class was introduced, and the tests compile fine. Ticket will stay open to solve the babel issue https://mattermost.atlassian.net/browse/MM-31410